development.env.example

```diff
@@ -21,3 +21,5 @@ HELLPER_VERIFICATION_TOKEN=YOUR_SLACK_VERIFICATION_TOKEN
 HELLPER_NOTIFY_ON_RESOLVE=true
 HELLPER_NOTIFY_ON_CLOSE=true
 FILE_STORAGE=google_drive
+HELLPER_PRODUCT_LIST=Product A;Product B;Product C
+TIMEZONE=America/Sao_Paulo
```

Comment:
I missed this environment variable in the documentation.
All environment variables must also be in:
* README.md
* app.json

Reply:
Done!